### PR TITLE
react todolist: Interactions

### PR DESCRIPTION
**Pull request by joskalenda**
#### Features :

- [ ] Implemented with react classes component
- [ ] add and remove todo from the list
- [ ] Update styling for the App

#### To Add

- [ ] Local storage
- [ ] validation
- [ ] Complete full website

### CODE REVIEW

File: src/components/TodoContainer.js [hunk 1]
```diff
@@ -1,37 +1,99 @@
...
+  //         return {
+
+  //           ...item, completed: !item.completed,
+  //         };
+  //       }
+  //       return item;
+  //     }),
+  //   }));
+  // };
+
```
Comment (orozCoding) [inline src/components/TodoContainer.js]: Remember to delete commented code once your finished in order to make your code cleaner 👍

File: src/components/InputTodo.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  };
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+    const { title, Addtodo } = this.state;
+    if (title.trim()) {
+      Addtodo.addTodoProps(title);
+      this.setState({ title: '' });
+    } else {
+      alert('Please write an item');
```
Comment (orozCoding) [inline src/components/InputTodo.js]: Remember that alerts, just as console logs, should be avoided in best practice. Once you finish the tutorial, remember to delete or change the alert. For example you could just add a `e.preventDefault()`.

**Review by orozCoding — COMMENTED** [review]
Hi @joskalenda 👋 



Good job so far!
I like the way the web-app looks and I believe you're doing a great job following the tutorial. I can tell you have tasks marked as done and some others marked as "to-do". I will be commenting on the tasks you have marked as done, and also will suggest some other changes you could add to your "to-do" list for this project.

# Required Changes ♻️

- [ ] One of the tasks you have marked as completed is "add and remove todo from the list", but neither adding nor removing is working properly in the app. I suggest you check the re-rendering of the app to see what's happening whenever the to-do list is updated.
- [ ]  Remember to delete commented code once you're finished in order to make your code cleaner 👍
- [ ]  Remember that alerts, just as console logs, should be avoided in best practice.
- [ ]  Once you finish the tutorial, remember to update your README file and the repositorie description so it is more related to the project 👌 
Check the comments under the review.

# Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

- [ ]  The default tasks you're working with, have some typos. As an optional suggestion, I believe it would be great if you install an orthography checker such as "Code Spell Checker By Street Side Software" for Visual Studio.
Check the comments under the review.

Cheers and Happy coding!👏👏👏

------

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by joskalenda** [comment]
Hey Angel, I did not know about `Code Spell Checker`, thanks for sharing that with me, I have corrected the code and I think now it is good to go... 😄

File: src/components/TodoList.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    const { listElem, handleChangeProps, removeTodoProps } = this.props;
+    return (
+      <ul>
+        {/* {List.listElem.map((items) => (<li key={items.id}>{items.title}</li>))} */}
```
Comment (orozCoding) [inline src/components/TodoList.js:15]: Remember to delete the comments 👌

File: src/components/TodoContainer.js [hunk 4]
```diff
@@ -1,37 +1,86 @@
...
+            ...todo,
+            completed: !todo.completed,
+          };
+        }
+        return todo;
+      }),
+    }));
+  };
+
+  // methode to remove the todo
```
Comment (orozCoding) [inline src/components/TodoContainer.js:47]: Little typo! It should be "method".

**Review by orozCoding — APPROVED** [review]
## **Hi @joskalenda,** 

## **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
 
_PD: I added a few optional suggestions to make it look even better!_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/TodoContainer.js (hunk 4 above)
Comment (joskalenda) [reply src/components/TodoContainer.js:47]: Thanks, @orozCoding Angle

**Pull request merged**